src/constants.js

```diff
@@ -1,6 +1,6 @@
 
 export const constants = {
-  SERVER_DOMAIN: process.env.REACT_APP_SERVER_DOMAIN,
+  SERVER_DOMAIN: 'https://api.beatcaps.io',
```

Comment:
is this ok? do we need to keep the env variable for anything else?

Reply:
yeah I use this one on a daily actually